cibyl/sources/zuul/apis/rest.puml

```diff
@@ -0,0 +1,60 @@
+@startuml
```

Comment:
What is this file? Is it for creating uml diagrams?

Reply:
It is meant for PlantUML, PyCharm has an editor for this tool and I took the chance. The file is a quick diagram of the file that I plan to implement. Something to get ideas together.

Reply:
Oh cool, I didn't know that. Maybe the docs folder is a good place to save them, or it has to be next to the code?

Reply:
No, it can be placed anywhere.